modules/weko-search-ui/weko_search_ui/static/js/weko_search_ui/app.js

```diff
@@ -334,12 +355,48 @@ function itemExportCtrl($scope, $rootScope, $http, $location) {
           }
         });
       })
+      exportBibtex=document.getElementById("export_format_radio_bibtex").checked
```

Comment:
Variable exportBibtex implicitly declared.
Please insert "var" or "let" for exportBibtex.